frontend/src/scenes/web-analytics/WebAnalyticsDashboard.tsx

```diff
@@ -223,6 +231,8 @@ export const WebTabs = ({
         ) : null,
     ].filter(isNotNil)
 
+    const isVisualizationToggleEnabled = [TileId.SOURCES, TileId.DEVICES, TileId.PATHS].includes(tileId)
```

Comment:
This is a prime place for a feature flag check. This feels very welcome and it feels like it's going to work, but we're big fans of FFs and I think we should add it here